Modules/posixmodule.c

```diff
@@ -14003,6 +14031,13 @@ all_ins(PyObject *m)
     if (PyModule_AddIntMacro(m, GRND_RANDOM)) return -1;
     if (PyModule_AddIntMacro(m, GRND_NONBLOCK)) return -1;
 #endif
+#ifdef HAVE_MEMFD_CREATE_SYSCALL
+    if (PyModule_AddIntMacro(m, MFD_CLOEXEC)) return -1;
+    if (PyModule_AddIntMacro(m, MFD_ALLOW_SEALING)) return -1;
+#ifdef MFD_HUGETLB
+    if (PyModule_AddIntMacro(m, MFD_HUGETLB)) return -1;
```

Comment:
Does it make sense to expose the flag without the supporting HUGETLB flags from ``HUGETLB_FLAG_ENCODE_64KB`` to ``HUGETLB_FLAG_ENCODE_16GB``?